react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -218,24 +218,18 @@ Only if they are is the isBundleUpToDate flag set to true. That flag is used by
         }
         inputs.files(inputFiles())
 
-        outputs.file("$buildAssetsFolder/$bundleName")
+        // We cannot use the bundle file itself as an output because it does not yet exist when this
+        // task finishes. Nevertheless, we have to declare something as an output because only tasks
+        // with outputs are run incrementally, so we're declaring a file that does not exist as the
+        // output. Since that file never exists, that "output" will always be considered "up-to-date".
```

Comment:
> Since that file never exists, that "output" will always be considered "up-to-date"

Heh, I find that counter-intuitive and probably shows my (low) gradle understanding so I wonder, do you have a link somewhere that explains this behavior?

Reply:
This was counter-intuitive to me too. This is [the forum post](https://discuss.gradle.org/t/unexpected-up-to-date-task-while-output-file-is-missing/5216/2) that clued me into the behavior:

> For many tasks the outputs are not explicit, so for each task that runs, Gradle records the outputs files in the provided dirs or file paths to scan. If none exist, and the tasks was successful, then the set of outputs for the task is null.